src/geyser/readme.md

```diff
@@ -7,6 +7,46 @@ opening a file-based pipe using the `mkfifo` syscall which is then
 written to like any other file. the key method used to setup 
 the pipes is `openPipe` in `src/geyser/core.zig`.
 
+## Usage
+
+while running, grafana stats will be available. the main binary code is in 
+`src/geyser/bin.zig`
+
+### benchmarking 
+
+we also have benchmarking to measure the throughput of geyser. you can run it using 
+
+```bash 
+zig build -Doptimize=ReleaseSafe
+./zig-out/bin/benchmark geyser
+```
+
+you can also benchmark an dummy reader
+
+```bash
+```bash
```

Comment:
i guess, this is a typo
```suggestion
```bash
```

Reply:
[0775da7](https://github.com/Syndica/sig/pull/279/commits/0775da7ac8a41c3bb16610db0544818e3b1e5764)